### ISSUE TITLE
[Tweak stopped working] - Red progress bar

### ISSUE THREAD

**Issue by KarmaBeast** [arichornlover/uYouEnhanced#506]
Suddenly the app crashes for me when opening videos. So disabling Sponsorblock seems have to have fixed it for now.

I also noticed that the Red progress bar tweak stopped working:

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/b60d1c88-82c1-4ade-b489-d662e6b5903c)

**Comment by KarmaBeast** [comment]
> Which YouTube version caused it to stop working?

For the past weeks I'm running this older version IPA by bhackel with Tap to seek. It worked fine all this time and just suddenly stopped working an hour ago. Out of blue...

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/bdd2e433-4e16-424e-b637-9e37a0b7afd3)

https://github.com/arichornlover/uYouEnhanced/assets/73917917/ecbe8f43-84ec-43f9-a995-ed24518d5d32

**Comment by eduedp** [comment]
It's also not working on latest standard uYouPlus

**Comment by KarmaBeast** [comment]
bhackel said in group chat "YouTube is testing something weird with the progress bar". I'm not sure.

**Comment by KarmaBeast** [comment]
I don't use any A/B feature
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/b39834c1-2979-4b65-8497-323bae5ae92c)

Tried testing with spoofer but that doesn't fix it either 

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/c186478f-c12d-470f-83f3-0ecb816299da)
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/9e1bc2b5-e9f3-4f0f-bfb2-220d7f54eb97)

**Comment by KarmaBeast** [comment]
Yeah I suppose it needs an update? It weirds to me that it break around the same time Sponsorblock started to crash the app

**Comment by KarmaBeast** [comment]
@arichornlover Nevermind when I updated to 19.13.1 now I noticed that it's working again! I don't know why it was suddenly broken on older version. But it works again with this version:

![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/f14d8f1e-91a0-4d25-88eb-965b30d2da42)

In this case should I close this issue, or keep it temporarily open for observation? (In case it somehow breaks again)

**Comment by eduedp** [comment]
I'm on 19.13.1 here and it's still not working. Did you change any settings lately? How did you make it work?

**Comment by KarmaBeast** [comment]
> I'm on 19.13.1 here and it's still not working. Did you change any settings lately? How did you make it work?

I changed nothing. Just updated to 13.1 and it suddenly works again! 🤷🏻‍♂️

**Comment by eduedp** [comment]
I’m using uYouEnhanced also. I just installed uYouPlus to test this bug

**Comment by bhackel** [comment]
Fixed in latest release, can be closed now
